Pod/Classes/IPDecimalNumberPadController.m

```diff
@@ -30,10 +34,11 @@ @implementation IPDecimalNumberPadController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+    [self setupCurrentAmount];
     [self setupBackgroundGradient];
+    [self setupAmountLabel];
     [self setupNumberPad];
-    [self setupAmountAndInstructionsLabels];
-    self.currentAmount = [[IPStringBackedDecimalValue alloc] init];
+    [self setNumberPadEdgeInsets:kIPDefaultNumberPadEdgeInsets];
```

Comment:
I would suggest here in view did load to simply 'load' the constraints based on the insets, and set the default inset in an init method instead(both initwithcoder and initwithnibname). Otherwise, the only way to override insets is to force the view to load first.